src/util.ts

```diff
@@ -655,11 +699,13 @@ export function thisExtensionPackage(): PackageJSON {
     const pkg: PackageJSON = thisExtension().packageJSON as PackageJSON;
 
     return {
-      name: pkg.name,
-      publisher: pkg.publisher,
-      version: pkg.version,
-      contributes: pkg.contributes
+        name: pkg.name,
```

Comment:
Nit: indentation seems changed, is this expected / wanted?

Reply:
yes. Should be 4 spaces